.github/workflows/generate_examples_coq.yml

```diff
@@ -0,0 +1,31 @@
+name: Generate Examples - Coq
+
+on: [pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/hacspec/hacspec_coq:8.16
+      options: --user root
+      credentials:
+        username: ${{ github.actor }}
```

Comment:
Should we just make the image public? Then this shouldn't be needed?

Reply:
Yes that is probably vest. This did not work regardless.

Reply:
I made it public and retriggered the job. We can drop the secret here then again I think.